From 5bc2fcbbaf547b2a684a95d2e044983fe7f640ee Mon Sep 17 00:00:00 2001
From: Bharat Kumar Gogada <bharat.kumar.gogada@xilinx.com>
Date: Mon, 13 Jan 2020 15:26:46 +0530
Subject: [PATCH 0849/1852] PCI: XDMA PL: Move to PCI version of mask and
 unmask MSI IRQ

commit c396eac212866000241de1d935507ab506cb5de9 from
https://github.com/Xilinx/linux-xlnx.git

The mask and unmask PCI MSI wrappers were removed.
Use the PCI version of these methods directly.

Signed-off-by: Bharat Kumar Gogada <bharat.kumar.gogada@xilinx.com>
State: not-upstreamable
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/pci/controller/pcie-xdma-pl.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/pci/controller/pcie-xdma-pl.c b/drivers/pci/controller/pcie-xdma-pl.c
index 82f3dcfb76cb..0ac9376ccfaf 100644
--- a/drivers/pci/controller/pcie-xdma-pl.c
+++ b/drivers/pci/controller/pcie-xdma-pl.c
@@ -469,10 +469,10 @@ static irqreturn_t xilinx_pcie_intr_handler(int irq, void *data)
 
 static struct irq_chip xilinx_msi_irq_chip = {
 	.name = "xilinx_pcie:msi",
-	.irq_enable = unmask_msi_irq,
-	.irq_disable = mask_msi_irq,
-	.irq_mask = mask_msi_irq,
-	.irq_unmask = unmask_msi_irq,
+	.irq_enable = pci_msi_unmask_irq,
+	.irq_disable = pci_msi_mask_irq,
+	.irq_mask = pci_msi_mask_irq,
+	.irq_unmask = pci_msi_unmask_irq,
 };
 
 static struct msi_domain_info xilinx_msi_domain_info = {
-- 
2.31.1

